logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/Message.kt

```diff
@@ -387,3 +391,12 @@ enum class AssetType {
 
 typealias ReactionsCount = Map<String, Int>
 typealias UserReactions = Set<String>
+
+sealed class RecipientFailure {
```

Comment:
I find it bit strange to have a `RecipientFailure` type which also have "no error" case. If it should also include the success case I'd suggest something along:

```
sealed class DeliveryStatus {
  data class PartialDelivery(...) : DeliveryStatus()
  object CompleteDelivery : DeliveryStatus()
}

Reply:
Makes total sense, this `DeliveryStatus` name suits better, thanks